spec/features/web_intake/requested_documents_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.feature "Client uploads a requested document" do
+  before do
+    allow_any_instance_of(Users::SessionsController).to receive(:idme_request).and_return(
+      user_idme_omniauth_callback_path(logout: "success")
+    )
+  end
```

Comment:
Why do we need to set up the logout for this test? Are we logging them out?

Reply:
nope! my bad!